osu.Game/OsuGame.cs

```diff
@@ -83,6 +85,8 @@ private Intro intro
 
         private Bindable<int> configSkin;
 
+        private Bindable<ScreenshotFormat> screenshotFormat;

```

Comment:
please don't pollute `OsuGame`. move *all* this new code out into a `ScreenshotManager` or similar.